topics/epigenetics/tutorials/hicexplorer/tutorial.md

```diff
@@ -0,0 +1,276 @@
+
+> ### {% icon hands_on %} Hands-on: hicMergeMatrixBins
+>
+> 1. **hicMergeMatrixBins** {% icon tool %}: Run hicMergeMatrixBins on the output from previous step setting the following parameter:
+>    - "Number of bins to merge" to `100`
+>
+> 2. **hicPlotMatrix** {% icon tool %}: Run hicPlotMatrix on the output from hicMergeMatrixBins adjusting the parameters:
+>    - "Plot title" to `Hi-C matrix for mESC`
+>    - "Plot per chromosome" to `True`
+>    - "Remove masked bins from the matrix" to `True`
+>    - "Chromosomes to include (and order to plot in)" to `chr2L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr2R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chrX`
+>
+>    > ### {% icon tip %} Tip: log1p
+>    >
+>    > Because of the large differences in counts found int he matrix, it is better to plot the counts using the *–log1p* option.
+>    {: .tip}
+>
+>    - **Plot the log1p of the matrix values**: True
+>
+{: .hands_on}
+
+# Correction of Hi-C matrix
+
+[hicCorrectMatrix](http://hicexplorer.readthedocs.io/en/latest/content/tools/hicCorrectMatrix.html#hiccorrectmatrix) corrects the matrix counts in an iterative manner. For correcting the matrix, it’s important to remove the unassembled scaffolds (e.g. NT_) and keep only chromosomes, as scaffolds create problems with matrix correction. Therefore we use the chromosome names (1-19, X, Y) here. **Important**: Use ‘chr1 chr2 chr3 etc.’ if your genome index uses chromosome names with the ‘chr’ prefix.
+
+Matrix correction works in two steps: first a histogram containing the sum of contact per bin (row sum) is produced. This plot needs to be inspected to decide the best threshold for removing bins with lower number of reads. The second steps removes the low scoring bins and does the correction.
+
+First we will re-run hicMergeMatrixBins on the output and set the bin size to 20.
+
+> ### {% icon hands_on %} Hands-on: Matrix diagnostic
+>
+> 1. **hicMergeMatrixBins** {% icon tool %}: Run hicMergeMatrixBins on the output setting the following parameter:
+>    - "Number of bins to merge" to `2`
+> 
+> 2. **hicCorrectMatrix** {% icon tool %}: Run hicCorrectMatrix on the output from previous step adjusting the parameters:
+>    - "Range restriction (in bp)" to `Diagnostic plot`
+>    - "Chromosomes to include (and order to plot in)" to `chr2L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr2R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3L`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chr3R`
+>    - "+ Insert Chromosomes to include (and order to plot in):" to `chrX`
+>
+{: .hands_on}
+

```

Comment:
Remove the first step starting at `First we will...`, and hicCorrectMatrix is step 1. We use the 10kb matrix.